package/suse-migration-sle15-activation-spec-template

```diff
@@ -46,6 +46,7 @@ install -D -m 755 99_migration \
     %{buildroot}/etc/grub.d/99_migration
 
 %post
+suse-migration-pre-checks
```

Comment:
I think this will not be enough because calling the tool in %post creates a dependency to the suse-migration-services package and suse-migration-sle15-activation has no dependency to it. What we need is a sub-package e.g ```suse-migration-pre-checks``` which falls out of suse-migration-services. That sub package must be submitted to the SUSE namespace to SLE12-SP4 and then we need a ```Requires: suse-migration-pre-checks``` in addition to the %post change here.

Makes sense ?

Reply:
Yes, I was not sure what was the best path/approach

I have added the changes for `suse-migration-pre-chekcs` package